### DISCUSSION TITLE
Tree: The Tree component's collapse functionality is not working

### DISCUSSION THREAD

**Discussion by rakeshvermak** [primefaces/community#2332] [PrimeReact]
Is it the default behavior of the PrimeReact Tree component that when we search for something and then collapse an expanded node, it doesn't stay collapsed?

I have checked it on https://primereact.org/tree/#filter, and it's not collapsing there either.

Is there any workaround for this issue?

**Comment by melloware** [comment]
Open a bug ticket in GitHub issues.
